### ISSUE TITLE
New error

### ISSUE THREAD

**Issue by darkairo** [DasKraken/CR-dl#10]
I apologize for bothering you so much with this, although I believe that I collaborate with the correct functioning of your code, now I got this:

>     Reflect.apply(handler, this, args);
>             ^
> 
> RuntimeException: Error: spawn ffmpeg ENOENT
>     at ChildProcess.<anonymous> (C:\snapshot\CR-dl\src\processVideo.js:0:0)
>     at ChildProcess.emit (events.js:182:13)
>     at Process.ChildProcess._handle.onexit (internal/child_process.js:235:12)
>     at onErrorNT (internal/child_process.js:406:16)
>     at process._tickCallback (internal/process/next_tick.js:63:19)

Do you know what may be happening? there is the tmp folder with 2 sub folders and a m3u8 file inside:

SubData
VodVidData
VodVid.m3u8

Regards!

**Comment by Wralui** [comment]
Make sure you have the last ffmpeg and not in conflit in the environment variable.

**Comment by DasKraken** [comment]
> I apologize for bothering you so much with this
Don't worry. I too want to have it working.

But well I see the issue. All of the last few issues were caused by VodVid.m3u8 beeing inside of the tmp folder, but also I don't want to leave it outside. In this case it's beause it changes the working directory but ffmpeg.exe isn't in there.
I looked a little bit more into it now, and it seems that ffmpeg works on windows the same way as on linux, but it doesn't handle ```\``` as line separator in input file path correctly, which caused #9.
